--- cppunit-config.in	2004-03-13 05:48:02.000000000 -0500
+++ cppunit-config.in.oden	2012-01-20 07:16:07.083890150 -0500
@@ -87,12 +87,5 @@
 fi
 
 if test "$echo_libs" = "yes"; then
-      if test @libdir@ != /usr/lib ; then
-            my_linker_flags="-L@libdir@"
-      fi
-      echo ${my_linker_flags} -lcppunit @LIBADD_DL@
-fi      
-
-
-
-
+      echo ${my_linker_flags} -lcppunit @LIBADD_DL@
+fi
